### PR TITLE
Support single `npm install` command from project root and without repo cloning by user

### PR DESCRIPTION
**Pull request by nunoguedelha**
Implements #95 .

### CODE REVIEW

**Comment by nunoguedelha** [comment]
@traversaro @RiccardoGrieco I'm still updating the README and will push it shortly, but you can already look at the code, and try the following steps for testing the simplified installation procedure:
- Got to an empty folder of your choice.
- run `npm install https://github.com/dic-iit/yarp-openmct.git --global-style true`.

**Comment by nunoguedelha** [comment]
@traversaro @RiccardoGrieco I've updated the README.

**Comment by RiccardoGrieco** [comment]
> * run `npm install https://github.com/dic-iit/yarp-openmct.git --global-style true`.

For testing this PR, the command should be pointing to the to-be-merged branch:

`npm install https://github.com/dic-iit/yarp-openmct.git#install/one-line-installer --global-style true`

File: README.md
Comment (RiccardoGrieco) [inline README.md:100]: I think that the comment concerning the YarpJS Node.js version restriction can be removed (as far as I understand also from the script), am I right?

Comment (nunoguedelha) [reply README.md:100]: As discussed, the issue is in the comment above this. Will fix.

Comment (nunoguedelha) [reply README.md:100]: Fixed README and `npm start`.

File: README.md
Comment (traversaro) [inline README.md:139]: Not blocking the PR, but I am not sure the long explanation in this place (the section is not for developers, but for naive users) is helping. I am afraid most users will be confused by this technical explanation.

**Review by traversaro — APPROVED** [review]

File: README.md
Comment (nunoguedelha) [reply README.md:139]: Good point, I will remove it in my next PR.

**Comment by nunoguedelha** [comment]
As soon as @RiccardoGrieco completes his installation test I'll merge the PR.

**Comment by RiccardoGrieco** [comment]
The server seems to start correctly with `npm start` from the module root.

However, I just put the robot in gazebo and started the server, but when I try to access the webpage it doesn't stop loading.

I did not run the `WalkingModule` since I have some configuration issue that I will try to solve the asap.
Is this infinite loading supposed to happen when some ports are missing or is there something else? Below a screenshot:

image

Also, the following line in the server terminal seems suspicious:

```sh
[OPEN-MCT STATIC SERVER] stdout: processedDefault.json requested!
```

**Comment by nunoguedelha** [comment]
> Is this infinite loading supposed to happen when some ports are missing or is there something else?

No, it should work even without the ports as we are using persistent ports on the visualizer server. What is happening is that when you have the `commonHostIPaddress` set to `'localhost'` in the `./config/default.json`, which resolves to `127.0.0.1` (the special internal IP address of the loopback virtual adapter), and for some reason, trying to load on the browser the address `http://127.0.0.1:8080` instead of `http://localhost:8080` won't work. This is an issue independent from the PR. Actually, the README tells to manually `commonHostIPaddress`set your machine address in `./config/default.json` if you wish the visualizer to be accessible by other machines. In any case, for a local test, `http://127.0.0.1:8080` should work. I'll open an issue for that.



> Also, the following line in the server terminal seems suspicious:
> 
> ```shell
> [OPEN-MCT STATIC SERVER] stdout: processedDefault.json requested!
> ```

This is normal. It's part of verbose runtime warnings. It was mostly useful during debug.

**Comment by nunoguedelha** [comment]
> I'll open an issue for that.

https://github.com/ami-iit/yarp-openmct/issues/98

**Comment by RiccardoGrieco** [comment]
> No, it should work even without the ports as we are using persistent ports on the visualizer server

Ok, I remember it this way but I was caught by the doubt 😆 .

> In any case, for a local test, `http://127.0.0.1:8080` should work

Indeed I tried, and it worked 👍🏻

File: README.md
Comment (RiccardoGrieco) [reply README.md:139]: Since this is introduced in this PR I think it is better to remove it before merging.

**Review by RiccardoGrieco — APPROVED** [review]

File: README.md
Comment (nunoguedelha) [reply README.md:139]: Sorry @RiccardoGrieco , I had missed your last comment, and you're right, for the sake of context. In this case, I had already included it in the next changes to be PRed.

**Pull request merged**
